doc/src/site/sphinx/deployment/load_mojo.rst

```diff
@@ -105,4 +105,23 @@ or in Python:
 
     from pysparkling.ml import *
     settings = H2OMOJOSettings(convertUnknownCategoricalLevelsToNa = True, convertInvalidNumbersToNa = True)
-    model = H2OMOJOModel.createFromMojo("prostate.mojo", settings)
\ No newline at end of file
+    model = H2OMOJOModel.createFromMojo("prostate.mojo", settings)
```

Comment:
qq: In Pysparkling, we return the correct type, right?

Reply:
Thanks to duck typing in Python it shouldn't matter if we call `H2OMOJOModel.createFromMojo` or `H2OTreeBasedSupervisedMOJOModel.createFromMojo`. I've added some tests [here](https://github.com/h2oai/sparkling-water/pull/1693/files#diff-17206c76b471d879b3ad5ad38e2283e8R165-R177)